scripts/dev/downloader.sh

```diff
@@ -271,6 +271,12 @@ downloader() {
                 EXTRA_ARGS+="--remove-on-error "
             fi
         fi
+        if curl -v | grep -q 'HTTP2'; then
+            CURL_SUPPORTS_HTTP2=1
+            CLOSE_CONNECTION=0
```

Comment:
http2 does this a different way, so no need to call as response has been generally port kept open and closes with timeout disregarding our header close so no need to send force close port message